lib/chewy/search/response.rb

```diff
@@ -33,6 +39,19 @@ def max_score
         @max_score ||= hits_root['max_score']
       end
 
+      # Response `errors` field. Returns `nil` if there is no error.
+      # @return [Hash, nil]
+      def errors
+        not_found_error = @body['not_found_error']
```

Comment:
Can you return early if body is nil here ?

Reply:
done